COMETwebapp/COMETwebapp.csproj

```diff
@@ -74,7 +74,6 @@
     <ItemGroup>
         <PackageReference Include="AntDesign" Version="0.14.1" />
         <PackageReference Include="BlazorStrap" Version="5.1.100" />
-        <PackageReference Include="CDP4.WEB.Common" Version="1.0.7" />
```

Comment:
Why do you remove the package?

Reply:
The project reference to common override it, and the common is not up to date